installVm-mac.sh

```diff
@@ -5,3 +5,21 @@ else
   echo "Hold on... installing multipass⏳"
   brew install multipass
 fi
+
+
+
+if (multipass info mos-vm)
+then
+  echo "The VM already exists"
+else
+  echo "Creating mos-vm"
+  multipass launch --name mos-vm
+fi
+
+
+if (multipass info mos-vm)
```

Comment:
You probably want to shell into the machine outside of the script. Shell into the vm is not IN the script that builds the vm since it would land you in the VM and just be stuck there.

Reply:
I'd remove this IF block and do something like this in the README

`bash installVM-mac.sh && multipass shell mos-vm`

Reply:
i dont understand the reasoning behind shelling into the machine outside the script? i am stupid..